frontend/lib/i18n/locales/nl/generic.json

```diff
@@ -1,28 +1,33 @@
 {
   "": "",
   "test": "test",
+  "activity_log": "Activiteitenlog",
   "account_configured": "Je account is ingesteld",
   "administrator": "Beheerder",
   "all_together": "Alles samen",
   "candidate": "Kandidaat",
   "close_message": "Melding sluiten",
   "coordinator": "Coördinator",
   "counted_number": "Geteld aantal",
+  "data_entry_clerc": "Invoerder",
+  "data_entry_stations": "Invoerstations",
```

Comment:
This is called `workstations` throughout the frontend code

Reply:
In `workstations.json` we have "Werkplekken beheren". So I think we need to choose between workstations/werkplekken en data entry stations/invoerstations.